tony-core/src/main/java/com/linkedin/tony/TonyClient.java

```diff
@@ -82,9 +82,13 @@
 
   private static final String APP_TYPE = "TENSORFLOW";
   private static final String ARCHIVE_SUFFIX = "tony_archive.zip";
-  private static final String CORE_SITE_CONF = YarnConfiguration.CORE_SITE_CONFIGURATION_FILE;
-  private static final String HADOOP_CONF_DIR = ApplicationConstants.Environment.HADOOP_CONF_DIR.key();
-  private static final String HDFS_SITE_CONF = "hdfs-site.xml";
+  private static final String CORE_SITE_CONF = Constants.CORE_SITE_CONF;
+  private static final String HADOOP_CONF_DIR = Constants.HADOOP_CONF_DIR;
+  private static final String HDFS_SITE_CONF = Constants.HDFS_SITE_CONF;
+  private static final String TONY_CONF_DIR = Constants.TONY_CONF_DIR;
```

Comment:
No need to redefine all these constants here if they're already defined in `Constants`. Just directly reference `Constants.XXX` when needed.

Reply:
+1

Reply:
👍